lib/widgets/atoms/section_tile.dart

```diff
@@ -2,9 +2,11 @@ import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:mm_flutter_app/constants/app_constants.dart';
 
+import 'notification_bubble.dart';
+
 class SectionTile extends StatefulWidget {
   final String title;
-  final Widget child;
+  final List<Widget> child;
```

Comment:
SectionTile was meant to be a generic widget that could be customized into specific sections. However, making *child* a list of widgets and then using it's length to show a notification bubble is very specific to the invitation section. If you want to have this logic be available to all sections it should be more generic, if not then we prefer to have this logic implemented into its own widget.

If the InvitationSection is the only that will use the SectionTile then feel free to merge it to avoid clutter.

Reply:
Hi Miguel,
Yes, I see you had used SectionTile for More Resources cards before. Currently, the only thing that is using SectionTile is the invitation section. When you say "merge it", do you mean I should define a "SectionTile" class in the invitation_section.dart file? and revert the section_tile.dart to its original form?
Let me know!

Reply:
I meant that we should either keep using SectionTile as a generic widget for all sections of the Home page, or make this logic you implemented here part of the InvitationSection, since none of the other sections would use it.

Reply:
The idea was to have repeating elements of Home page sections encapsulated in the SectionTile, so that we don't have to define it multiple times. This is similar to how we used a template widget for the signup flow.

Reply:
"make this logic you implemented here part of the InvitationSection" I will do the following, since I do not see elements other widgets needing SectionTile